encodings/fastlanes/src/bitpacking/compress.rs

```diff
@@ -54,22 +69,33 @@ pub unsafe fn bitpack_encode_unchecked(
     array: PrimitiveArray,
     bit_width: u8,
```

Comment:
instead of making `try_new` unsafe, why don't we just compute_min if signed. We either have or will compute it, and it's memoized